module/VuFind/src/VuFind/ILS/Driver/AlmaFactory.php

```diff
@@ -69,7 +69,6 @@ public function __invoke(
         // passed in as options):
         $driver = new $requestedName(
             $container->get(\VuFind\Date\Converter::class),
-            $container->get(\VuFind\Config\PluginManager::class),
```

Comment:
Making this change turns AlmaFactory into a copy-and-paste of DriverWithDateConverterFactory. I think we can delete this factory and change the configuration in the PluginManager.

Reply:
Awesome, one _less_ file! Done.